references/recaptcha-enterprise/README.md

```diff
@@ -0,0 +1,186 @@
+# reCAPTCHA Enterprise & Apigee X/hybrid
+
+[Google reCAPTCHA Enterprise](https://cloud.google.com/recaptcha-enterprise)
+can be leveraged by Apigee in order to protect backend systems from fraudulent
+client side activities like **bots**.
+A reCAPTCHA token is silentely and periodically retrieved by a client app and
+transmitted to an Apigee runtime when an API is invoked.
+
+## How it works?
+
+The sharedflow configuration - that is proposed through this Apigee devrel
+reference - is able to get a reCAPTCHA token validation status and a risk score
+from the Google reCAPTCHA Enterprise assessment endpoint.
+The risk score is a decimal value between ```0.0``` and ```1.0```.
+
+The score ```1.0``` indicates that the interaction poses low risk and is very
+likely legitimate, whereas ```0.0``` indicates that the interaction poses high
+risk and might be fraudulent. Between both extremes, the sharedflow's
+processing decides if an API invocation must be rejected or not.
+The minimum score to consider an API call as legitimate is set at ```0.6```:
+this value is configurable.
+
+## Apigee runtime options
+
+The reCAPTCHA enterprise reference can be deployed on both Apigee X and
+hybrid.
+
```

Comment:
Up to you if you want to mention that this would also work with Edge if the Service Account token is obtained though the proxy.